web/package.json

```diff
@@ -71,5 +73,8 @@
     "eslint": "^8.48.0",
     "eslint-config-next": "^14.1.0",
     "prettier": "2.8.8"
+  },
+  "overrides": {
+    "react-is": "^19.0.0-rc-69d4b800-20241021"
```

Comment:
needed to support shadcn charts with nextjs 15 

https://ui.shadcn.com/docs/react-19

Reply:
a bit messy since we haven't switched to use react19, but it seems to work 🤷

Reply:
without it, the charts are empty